### PR TITLE
fix modal bug, edit profile, my profile, change url route .. works

### PR DESCRIPTION
**Pull request by eliphazbouye**

### CODE REVIEW

File: resources/views/users/client/home.blade.php [hunk 1]
```diff
@@ -56,7 +56,7 @@
           </div>
         </div>
       </div>
-
+      {{-- {{ $projects->onEachSide(5)->links() }} --}}
```
Comment (daviidy) [inline resources/views/users/client/home.blade.php:59]: Please delete all the commented code on your files

File: resources/views/welcome.blade.php [hunk 2]
```diff
@@ -6,7 +6,8 @@
             <div class="col-md-12">
               <div class="text-center position-relative header-text">
                 <h1 class="display-1 font-weight-bold text-white">Swipe Right</h1>
-                <button class="btn mt-4 text-white rounded-pill px-5 font-weight-bolder" data-toggle="modal" data-target="#registerModal">Créer un compte</button>
+                <button class="btn mt-4 text-white rounded-pill px-5 font-weight-bolder">  <a href="{{ route('register') }}" style="color: white ">Créer un compte</a> </button>
+                {{-- <button class="btn mt-4 text-white rounded-pill px-5 font-weight-bolder" data-toggle="modal" data-target="#registerModal">Créer un compte</button> --}}
```
Comment (daviidy) [inline resources/views/welcome.blade.php:10]: Please delete all the commented code on your files

**Review by daviidy — COMMENTED** [review]
- [ ] Avoid the fork of the repo. Create directly a feature branch on your own. Then push the work you've done on your feature branch to github. Finally request a review. Because I cannot pull your work on my computer to test the feature

File: resources/views/users/client/home.blade.php (hunk 1 above)
Comment (eliphazbouye) [reply resources/views/users/client/home.blade.php:59]: this code comment will have their importance

File: resources/views/welcome.blade.php (hunk 2 above)
Comment (eliphazbouye) [reply resources/views/welcome.blade.php:10]: this code comment will have their importance

**Pull request closed without merging**